### PR TITLE
Allow building the archive using the docker-build script

### PR DESCRIPTION
**Pull request by frederikrothenberger**
This allows to build the archive using docker the same way as II.

### CODE REVIEW

File: scripts/docker-build [hunk 1]
```diff
@@ -70,31 +78,48 @@ function build() {
...
+        --archive)
+            CANISTER+=("archive")
+            shift
+            ;;
         *)
             echo "ERROR: unknown argument $1"
             usage
             echo
-            echo "Use 'release --help' for more information."
+            echo "Use 'docker-build --help' for more information"
```
Comment (nmattia) [inline scripts/docker-build:111]:
```suggestion
            echo "Use '$0 --help' for more information"
```

might as well prevent future copy pasta

File: scripts/docker-build [hunk 2]
```diff
@@ -70,31 +78,48 @@ function build() {
...
             exit 1
             ;;
     esac
 done
 
-build
+# build everything by default
```
Comment (nmattia) [inline scripts/docker-build:117]: this only builds II though, right?

Comment (frederikrothenberger) [reply scripts/docker-build]: Ah, yes. I have also fixed the same error in the `build` script.

File: scripts/docker-build [hunk 3]
```diff
@@ -70,31 +78,48 @@ function build() {
...
-    cp "$tmp_outdir/internet_identity.wasm" .
+    cp "$tmp_outdir/$canister.wasm" .
 
     echo "Removing $tmp_outdir"
     rm -rf "$tmp_outdir"
 }
 
 # ARGUMENT PARSING
 
+CANISTER=()
```
Comment (nmattia) [inline scripts/docker-build:89]: nit: this should probably be `CANISTERS`

**Review by nmattia — APPROVED** [review]

**Pull request merged**